addons/l10n_ar_base/data/res_country_data.xml

```diff
@@ -0,0 +1,1098 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model='res.country' forcecreate="False" id='base.ad'>
```

Comment:
What do you mean that this might be overkill?
We think it's better to keep the  forcecreate as we have found many issues while upgrading or installing the module in databases where the customer has deleted some countries. Of course there is no need to delete country but as user is able to do that we need to be prepared.